ddl/backfilling_scheduler.go

```diff
@@ -110,6 +111,8 @@ func (b *txnBackfillScheduler) setupWorkers() error {
 }
 
 func (b *txnBackfillScheduler) sendTask(task *reorgBackfillTask) {
+	b.taskMaxID++
```

Comment:
Why reallocate the task ID?

Reply:
Because it needs to be unique during the lifetime of the DDL job, instead of a task batch.